### PR TITLE
Moved maven dependency to maven wrapper

### PR DESCRIPTION
**Pull request by lasbreyn**
Moved maven dependency to maven wrapper

### CODE REVIEW

File: idp-webapp-overlay/etc/jetty/jetty.xml [hunk 1]
```diff
@@ -78,7 +78,7 @@
     <!-- =========================================================== -->
     <New id="httpConfig" class="org.eclipse.jetty.server.HttpConfiguration">
       <Set name="secureScheme"><Property name="jetty.httpConfig.secureScheme" default="https" /></Set>
-      <Set name="securePort"><Property name="jetty.httpConfig.securePort" deprecated="jetty.secure.port" default="443" /></Set>
```
Comment (mmoayyed) [inline idp-webapp-overlay/etc/jetty/jetty.xml:81]: Dont change the default. Revert this change please.

**Comment by mmoayyed** [comment]
There is a blank idp-webapp-overlay/etc/jetty/start.ini . What does that do?

**Comment by lasbreyn** [comment]
That is meant to be the config file for the ports.
I am not done with it yet.

Lasbrey Nwachukwu
Phone: 248.910.7395
Fax: 928.962.7935
Email: lasbreyn@gmail.com
LinkedIn: http://www.linkedin.com/pub/lasbrey-nwachukwu/13/283/8b4

**Comment by mmoayyed** [comment]
OK Cool. Lets pull that file out of this pull here then, since it doesnt belong here. 

Start your changes on a new branch.

**Comment by mmoayyed** [comment]
For the jetty config, that is.
